glide.yaml

```diff
@@ -0,0 +1,332 @@
+package: github.com/coreos/mantle
+import:
+- package: bitbucket.org/ww/goautoneg
+  version: 75cd24fc2f2c2a2088577d12123ddee5f54e0675
+- package: github.com/Azure/azure-sdk-for-go
+  version: f8b0607613f19ae9509b5ed6fbfda56caf06d59d
+  subpackages:
+  - management
+  - management/location
+  - management/storageservice
+  - storage
+- package: github.com/Microsoft/azure-vhd-utils-for-go
+  version: 070db2d701a462ca2edcf89d677ed3cac309d8e8
+  subpackages:
+  - upload
+  - upload/concurrent
+  - upload/metadata
+  - upload/progress
+  - vhdcore
+  - vhdcore/bat
+  - vhdcore/block
+  - vhdcore/block/bitmap
+  - vhdcore/common
+  - vhdcore/diskstream
+  - vhdcore/footer
+  - vhdcore/header
+  - vhdcore/header/parentlocator
+  - vhdcore/reader
+  - vhdcore/validator
+  - vhdcore/vhdfile
+  - vhdcore/writer
+- package: github.com/ajeddeloh/go-json
+  version: 73d058cf8437a1989030afe571eeab9f90eebbbd
+- package: github.com/ajeddeloh/yaml
+  version: 1072abfea31191db507785e2e0c1b8d1440d35a5
+- package: github.com/alecthomas/units
+  version: 2efee857e7cfd4f3d0138cc3cbb1b4966962b93a
+- package: github.com/aws/aws-sdk-go
+  version: 40bc7761a9f06daa574d20f2ad5454db02a05953
```

Comment:
I'd rather use version tags for things, like the aws-sdk, where there are suitable tags available.